### PR TITLE
JSTL - AdminAccountDetailsUiTest

### PR DESCRIPTION
**Pull request by bamboo3250**
Added HTML verification for the test

### CODE REVIEW

**Comment by bamboo3250** [comment]
Ah, the reason for the JSTL tag is that we will use this for the JSTL migration.

File: src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java [hunk 1]
```diff
@@ -70,7 +73,7 @@ public void testRemoveFromCourseAction(){
         detailsPage.clickRemoveStudentFromCourse(courseId)
             .verifyStatus(Const.StatusMessages.INSTRUCTOR_REMOVED_FROM_COURSE);
         assertNull(BackDoor.getStudent(courseId, "AAMgtUiT.instr2@gmail.com"));
-    
+        detailsPage.verifyHtml("/adminAccountDetailsRemoveStudent.html");
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java]: You can change this to verifyHtmlMainContent. I think one verifyHtml is enough for one  Ui test

File: src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java [hunk 2]
```diff
@@ -62,6 +62,9 @@ public void testRemoveFromCourseAction(){
         
         detailsPage.clickRemoveInstructorFromCourse(courseId)
             .verifyStatus(Const.StatusMessages.INSTRUCTOR_REMOVED_FROM_COURSE);
+        
+        detailsPage.verifyHtml("/adminAccountDetailsRemoveInstructor.html");
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java]: Maybe a html comparison here is a bit superfluous. Is there anything in particular you would like to test here?

File: src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java [hunk 3]
```diff
@@ -70,7 +73,7 @@ public void testRemoveFromCourseAction(){
         detailsPage.clickRemoveStudentFromCourse(courseId)
             .verifyStatus(Const.StatusMessages.INSTRUCTOR_REMOVED_FROM_COURSE);
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java]: Not related to your changes, but this status message looks wrong

**Comment by kanghj** [comment]
Some minor changes requested

File: src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java (hunk 3 above)
Comment (bamboo3250) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminAccountDetailsPageUiTest.java]: Oh right, I didn't notice that :open_mouth:

**Comment by bamboo3250** [comment]
fixed

**Pull request merged**